### ISSUE TITLE
NVDA does not read in Geekbench 6.4

### ISSUE THREAD

**Issue by mzanm** [nvaccess/nvda#17892]
Related: #16251

### Steps to reproduce:
1. Download and install Geekbench 6.
2. Launch the utility and press alt to focus the ribbon menu.

### Actual behavior:

NVDA does not speak the focus, trying to navigate with tab NVDA is silent, and the following error appears in the log:

```
ERROR - comtypes._comobject.call_without_this (16:23:13.975) - Dummy-3 (7736):
Exception in IUIAutomationFocusChangedEventHandler.HandleFocusChangedEvent implementation:
Traceback (most recent call last):
  File "comtypes\_comobject.pyc", line 154, in call_without_this
  File "UIAHandler\__init__.pyc", line 847, in IUIAutomationFocusChangedEventHandler_HandleFocusChangedEvent
  File "UIAHandler\__init__.pyc", line 1438, in isNativeUIAElement
  File "UIAHandler\__init__.pyc", line 1275, in isUIAWindow
  File "UIAHandler\__init__.pyc", line 1168, in _isUIAWindowHelper
ValueError: invalid literal for int() with base 10: '6,4,0,0'
```

### Expected behavior:

NVDA reads the controls in the ribbon normally.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2024.4.2

#### Windows version:
Windows 11 24H2 26100.3476

#### Name and version of other software in use when reproducing the issue:
Geekbench6 6.4.0

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, latest alpha. The same issue occurs.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.
